app/import_functions_test.go

```diff
@@ -2681,8 +2681,8 @@ func TestImportImportEmoji(t *testing.T) {
 	err := th.App.ImportEmoji(&data, true)
 	assert.NotNil(t, err, "Invalid emoji should have failed dry run")
 
-	result := <-th.App.Srv.Store.Emoji().GetByName(*data.Name)
-	assert.Nil(t, result.Data, "Emoji should not have been imported")
+	emoji, err := th.App.Srv.Store.Emoji().GetByName(*data.Name)
```

Comment:
If you are ignoring the error just use `_` to be explicit about it. Other option (I think is better) is to use `require.NotNil(t, err)` or `require.Nil(t, err)` depending of what you really expect there.